starknet_py/tests/e2e/docs/code_examples/test_account.py

```diff
@@ -36,15 +36,15 @@ async def test_execute(account, contract_address):
         max_fee=int(1e15),
     )
     # or
-    # docs-end: execute
+    # docs-end: execute_v1
```

Comment:
Could we also add an example for `execute_v3`? 🙏 It can use the same call as here.